build.sh

```diff
@@ -639,12 +640,30 @@ build_spidermonkey() {
 }
 
 
-fetch_dacapo_jar() {
-    echo "\n===> Download DaCapo .jar file\n"
+build_dacapo() {
+    echo "\n===> Build DaCapo\n"
 
     if [ -f "${HERE}/extbench/dacapo-9.12-bach.jar" ]; then return; fi
 
-    wget "http://downloads.sourceforge.net/project/dacapobench/9.12-bach/dacapo-9.12-bach.jar?r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Fdacapobench%2Ffiles%2F&ts=1474888492&use_mirror=freefr" -O ${HERE}/extbench/dacapo-9.12-bach.jar  || exit $?
+    cd ${wrkdir}
```

Comment:
Can we add a brief comment saying why we build it and then fetch a binary distribution. You could even copy/paste from the commit message which explains nicely.